### DISCUSSION TITLE
刚进绳网

### DISCUSSION THREAD

**Discussion by ChenYanxuan579** [share121/inter-knot#4771] [灌水]
发帖和回复都只能在网页版吗，app端只能浏览吗

**Comment by Yotusgi** [comment]
是这样的<img src="https://bbs-static.miyoushe.com/static/2024/07/31/ed2b65b6899fc16528b1d10adfcb9a38_8705924363701631267.png" alt="云璃 白眼" width="75" >

**Comment by ztjh** [comment]
现在不一样了

**Comment by aaahai00** [comment]
我这里会跳转到GitHub app回复，这一条就是这样发的

**Comment by 1580-D181** [comment]
手机也行啊😆

**Comment by hpp326** [comment]
app可以啊，回复和发帖会跳转到github，就可以发送了

**Comment by yanER582136479** [comment]
这段话能发出去吗？
